app/templates/gulpfile.js

```diff
@@ -473,17 +482,35 @@ function serve(isDev, specRunner) {
     });
 }
 
-function getNodeOptions(isDev) {
-  var port = args['app-port'] || process.env.PORT || config.defaultPort || 9070;
+/**
+ * Determine whether the enviornment requires dev mode'
+ * @param {String} env - 'local', 'dev', or 'prod'
+ * @return {Boolean} - true if env==='local'
+ */
+function isDevMode(env) {
+  return env === 'local';
+}
+
+function getNodeOptions(env) {
+  var envJson;
+  var envFile = 'gulp-' + env + '.json';
+    try {
+      envJson = require(envFile);
+    }
+    catch (e) {
+      envJson = {};
+      console.log('Info: ' + envFile + ' - may not exist. Did you mean to create one?');
```

Comment:
A different project was using a similar pattern for a `local.gradle` file. If missing it printed:

"Couldn't find local.gradle; you can create this gitignore'd file to override properties if necessary"

Maybe print a similar message in our case?